### PR TITLE
feat: prepare supporting runtime

### PR DESCRIPTION
**Pull request by underfin**
### Description

related https://github.com/rolldown-rs/rolldown/issues/35

### Test Plan

no need.

---

### CODE REVIEW

File: crates/rolldown/src/bundler/graph/graph.rs [hunk 1]
```diff
@@ -13,6 +13,7 @@ pub struct Graph {
   pub entries: Vec<(Option<String>, ModuleId)>,
   pub sorted_modules: Vec<ModuleId>,
   pub symbols: Symbols,
+  pub runtime: ModuleId,
```
Comment (underfin) [inline crates/rolldown/src/bundler/graph/graph.rs]: The runtime module put into `modules` of graph and use `runtime` to refrence it, but maybe is good to put it to `runtime`.

**Comment by underfin** [comment]
!bench

**Comment by hyf0** [comment]
Would you mind holding this PR before #27 being merged? I need to apply some changes to this PR after #27.

**Comment by underfin** [comment]
ok

File: crates/rolldown/src/bundler/module_loader/module_task.rs [hunk 2]
```diff
@@ -50,7 +51,11 @@ impl ModuleTask {
     let mut builder = ModuleBuilder::default();
     tracing::trace!("process {:?}", self.path);
     // load
-    let source = tokio::fs::read_to_string(self.path.as_ref()).await?;
+    let source = if RUNTIME_PATH == self.path.as_ref() {
```
Comment (hyf0) [inline crates/rolldown/src/bundler/module_loader/module_task.rs:54]: I'm intending to not include `runtime` module as the part of making module graph. In this way, we don't need to write such code.

Another way is to have a special `ModuleTaskForRuntime` to deal with the special logic of runtime module.

I haven't think through which way is better. Let's change it later.

**Pull request merged**